js/main.js

```diff
@@ -5,3 +5,9 @@ $('[data-action="run"]').click(function() {
 $('[data-action="clear-console"]').click(function() {
     window.clearConsole();
 });
+
+$('#console').bind('keydown', 'ctrl+l', function() {
+    window.clearConsole();
+});
```

Comment:
does this handle `cmd+l` for mac users?

Reply:
Mac users also clear their terminals/consoles via Ctrl+L ✅

Reply:
okay perfect :+1: ( **PS:** i have never been a mac user before :joy: )

Reply:
😂

Reply:
You guys are missing out on a lot :see_no_evil:

fyi, to clear terminal, mac users use command-k (`⌘ + K`)

Reply:
:see_no_evil: :see_no_evil: :see_no_evil:

Reply:
@desoukya I've been using a Mac for 3 months now, and I've been clearing the terminal using Ctrl+L the entire time - I thought it was the only key binding for this 😂 Thanks for the tip! 🙈

Reply:
I'm glad to hear you have converted to the light side :tada:

I'm glad to hear CTRL-L works! Maybe I have a custom mapping on my machine. 

Thanks for your contribution; it's always nice to see a familiar face :slightly_smiling_face: